File: src/Vincent OS/Utilisateur.cs
using System;

namespace Vincent_OS
{
    public partial class Utilisateur
    {
        public Utilisateur()
        {
            InitializeComponent();
        }

        private void Utilisateur_Load(object sender, EventArgs e)
        {
            Bureau bureau = new Bureau();
            Bureau2 bureau2 = new Bureau2();
            Login login = new Login();
            if (bureau.Visible == true)
            {
                if (login.Label2.Text == "Superutilisateur")
                {
                    Label2.Text = "Superutilisateur";
                }
                else
                {
                    Label2.Text = login.Label2.Text;
                }
            }
            else if (bureau2.Visible == true)
            {
                if (login.Label3.Text == "Invité")
                {
                    Label2.Text = "Invité";
                }
                else
                {
                    Label2.Text = login.Label3.Text;
                }
            }
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            Bureau bureau = new Bureau();
            Bureau2 bureau2 = new Bureau2();
            Login login = new Login();
            Login2bureau login2bureau = new Login2bureau();
            Login2bureau2 login2bureau2 = new Login2bureau2();
            Paramètres paramètres = new Paramètres();
            if (bureau.Visible == true)
            {
                login.Label2.Text = TextBox1.Text;
                login2bureau.Text = "Bienvenue sur Vincent OS : " + TextBox1.Text;
                bureau.Text = "Bureau - Session " + TextBox1.Text;
                bureau.Label3.Text = TextBox1.Text;
                paramètres.Label7.Text = "Utilisateur : " + TextBox1.Text;
                Label2.Text = TextBox1.Text;
            }
            else if (bureau2.Visible == true)
            {
                login.Label3.Text = TextBox1.Text;
                login2bureau2.Text = "Bienvenue sur Vincent OS : " + TextBox1.Text;
                bureau2.Text = "Bureau - Session " + TextBox1.Text;
                bureau2.Label3.Text = TextBox1.Text;
                paramètres.Label7.Text = "Utilisateur : " + TextBox1.Text;
                Label2.Text = TextBox1.Text;
            }
        }
    }
}

File: src/Vincent OS/Utilisateur.vb
Public Class Utilisateur

    Private Sub Utilisateur_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        If Bureau.Visible = True Then
            If Login.Label2.Text = "Superutilisateur" Then
                Me.Label2.Text = "Superutilisateur"
            Else
                Me.Label2.Text = Login.Label2.Text
            End If
        ElseIf Bureau2.Visible = True Then
            If Login.Label3.Text = "Invité" Then
                Me.Label2.Text = "Invité"
            Else
                Me.Label2.Text = Login.Label3.Text
            End If
        End If
    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        If Bureau.Visible = True Then
            Login.Label2.Text = Me.TextBox1.Text
            Login2bureau.Text = "Bienvenue sur Vincent OS : " + Me.TextBox1.Text
            Bureau.Text = "Bureau - Session " + Me.TextBox1.Text
            Bureau.Label3.Text = Me.TextBox1.Text
            UAC.Label2.Text = Me.TextBox1.Text
            Paramètres.Label7.Text = "Utilisateur : " + Me.TextBox1.Text
            Label2.Text = TextBox1.Text
        ElseIf Bureau2.Visible = True Then
            Login.Label3.Text = Me.TextBox1.Text
            Login2bureau2.Text = "Bienvenue sur Vincent OS : " + Me.TextBox1.Text
            Bureau2.Text = "Bureau - Session " + Me.TextBox1.Text
            Bureau2.Label3.Text = Me.TextBox1.Text
            Paramètres.Label7.Text = "Utilisateur : " + Me.TextBox1.Text
            Label2.Text = TextBox1.Text
        End If
    End Sub
End Class